### ISSUE TITLE
unused-method-argument doesn't pick up scoped arguments.

### ISSUE THREAD

**Issue by ocssor** [ditinc/linter-cflint#18]
The rule for unused-method-arguments is a good one, however, it doesn't pick up if you've used the argument by scoping it, e.g. it will say 'foo' is unused even though 'arguments.foo' is used in the function.

**Comment by ryaneberly** [comment]
Can you provide a code sample.

**Comment by ocssor** [comment]
@ryaneberly sure:

```
<cffunction name="test" output="false" returntype="text">
    <cfargument name="foo" type="string" default="" hint="test">

    <cfset testFunction(arguments.foo)>

</cffunction>
```

Creates an 'unused-method-argument' warning.

**Comment by sjmatta** [comment]
@ryaneberly Is this the same as https://github.com/cflint/CFLint/issues/152?

**Comment by ryaneberly** [comment]
yes, and it is fixed. 0.7.2 I think.

**Comment by sjmatta** [comment]
It seems to still be an issue in 0.7.3, at least in `linter-cflint` for whatever reason, but I need to confirm. Stay tuned!

**Comment by sjmatta** [comment]
Seems to be resolved in linter-cflint 0.13.0
